x/bank/keeper/keeper.go

```diff
@@ -3,6 +3,7 @@ package keeper
 import (
 	"fmt"
 
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
```

Comment:
We do not want the bank module to have any dependencies on the gov module. That's why I said to just hard-code our own copy of the string in here.